### ISSUE TITLE
Credit card update() not working

### ISSUE THREAD

**Issue by Idorobots** [paypal/PayPal-Java-SDK#106]
Analogously to #46, `CreditCard.update()` sends [a JSON representation](https://github.com/paypal/PayPal-Java-SDK/blob/v1.2.4/rest-api-sdk/src/main/java/com/paypal/api/payments/CreditCard.java#L499) of the `CreditCard` instead of the [expected](https://developer.paypal.com/docs/api/#update-a-stored-credit-card) payload, for example:

```json
[
    {
        "op": "replace",
        "path": "/billing_address",
        "value": {
            "line1": "111 First Street",
            "city": "Saratoga",
            "country_code": "US",
            "state": "CA",
            "postal_code": "95070"
        }
    }
]
```

**Comment by juwlee** [comment]
Fixed in `v1.2.5`. Thanks for reporting @Idorobots

**Comment by Idorobots** [comment]
Great! Thanks. :+1:
